### PR TITLE
Rename module

### PR DESCRIPTION
**Pull request by mattias-p**

### CODE REVIEW

**Comment by matsduf** [comment]
I approve the change with the following change. In the bottom of README.md there is the following line:

> /Calle Dybedahl calle@init.se, 12 February 2015

Either remove it or add text that it has been updated and the date of update.

**Comment by mattias-p** [comment]
@matsduf can you take a look at the last changes?

**Comment by pawal** [comment]
If you remove names like that, please consider adding a CONTRIUTORS file, naming the people that has contributed to the software. This is a common courtesy in open source projects.

**Comment by mattias-p** [comment]
Good point, @pawal. I just thought his name didn't fit in that particular place since he's not the main contact for the module anymore. It wasn't my intention to erase him. Hope this contributors file will put things right.

**Comment by mattias-p** [comment]
Oh, this is against the master branch, by the way. Should I make a new PR against the develop branch? I'm not sure how much of a difference it makes. The master branch is already out of sync with the latest release with 12 commits.

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Calle Dybedahl <calle@init.se>
```
Comment (matsduf) [inline CONTRIBUTORS.txt]: Shouldn't the first line here point at NL Netlabs' LDNS contributors?

**Comment by mattias-p** [comment]
Rebased to current master and added a fix for a class name truncation that was introduced by the renaming.

**Comment by mattias-p** [comment]
@matsduf, do you want to take a last look or should I just merge this?

**Pull request merged**